File: TUKulinarium_API/Helpers/HelperClasses/Dish.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TUKulinarium_API.Helpers.BaseClasses
{
    public class Dish
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int DishId { get; set; }
        public string DishName { get; set; }
        public decimal DishPrice { get; set; }
    }
}

File: TUKulinariumClient/src/app/models/Dish.ts
export class Dish {
  DishId: number;
  DishName: string;
  DishPrice: number;
  DishIngredients?: string[];

  constructor(id: number, name: string, price: number, ingredients: string[]) {
    this.DishId = id;
    this.DishName = name;
    this.DishPrice = price;
    this.DishIngredients = ingredients;
  }
}
